mad-dog/src/tests/test_userReplicaSetManager.js

```diff
@@ -34,14 +34,16 @@ const verifyUserReplicaSetStatus = async (
     let usrQueryInfo = await getUser(libs, userId)
 
     // Validate the latest updater indexed into discovery node matches expected value
-    let replicaSetUpdateSigner = replicaSetUpdaterAddress
+    let queriedReplicaSetUpdateSigner = usrQueryInfo.replica_set_update_signer.toLowerCase()
+
+    let validReplicaSetUpdateSigner = [replicaSetUpdaterAddress]
     if (!replicaSetUpdateSigner) {
-      replicaSetUpdateSigner = libs.getWalletAddress()
+      const signers = await getAllSigners(usrQueryInfo)
```

Comment:
the more I think about it, this fn should probably not even need a param to pass in a wallet, and always check whether or not the rset signer is the user itself, or one of the rset nodes cus that should always be the case. 

for simplicity's sake, it might be a good idea to always append the `signers`

Reply:
Well, if you wanted to compare against a specific signer, then I think it'd be useful

Reply:
yeah that's fair. fine by me